pattern.go

```diff
@@ -196,12 +197,33 @@ func readSpecialPattern(pb *patternBuild, valsIn []typedVal) (pathVals []typedVa
 		pathVals, err = readExistsSpecial(pb, pathVals)
 	case "shellstyle":
 		pathVals, err = readShellStyleSpecial(pb, pathVals)
+	case "prefix":
+		pathVals, err = readPrefixSpecial(pb, pathVals)
 	default:
 		err = errors.New("unrecognized in special pattern: " + tt)
 	}
 	return
 }
 
+func readPrefixSpecial(pb *patternBuild, valsIn []typedVal) (pathVals []typedVal, err error) {
+	t, err := pb.jd.Token()
+	if err != nil {
+		return
+	}
+	pathVals = valsIn
```

Comment:
What's the purpose of this assignment? This is not a (deep) copy.

Reply:
Naked returns at work. More on this below

Reply:
Ok, besides the named (naked) returns, you don't care that this is a shall copy of `valsIn` here, i.e. immutability for the source slice is not required, correct?